documentation/sphinx/source/release-notes/release-notes-710.rst

```diff
@@ -4,6 +4,10 @@
 Release Notes
 #############
 
+7.1.20
+======
+* Introduce STORAGE_SERVER_REBOOT_ON_IO_TIMEOUT knob to recreate SS at io_timeout errors. `(PR #8122) <https://github.com/apple/foundationdb/pull/8122>`_
```

Comment:
This should be put in 7.1.22, 7.1.21 has already been released: https://github.com/apple/foundationdb/blob/release-7.1/documentation/sphinx/source/release-notes/release-notes-710.rst

Reply:
ah, I didn't see a release note on 7.1.22. Let me change mine.

Reply:
changed